src/components/ChatLog.jsx

```diff
@@ -0,0 +1,18 @@
+import * as PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+import {useState} from 'react';
+
+const ChatLog = ({entries}) => {
+  const [numOfLikes, setLikes] = useState(entries?.filter(i => i.liked === true)?.length);
+  return <div>
+    <div>
+      {`${numOfLikes} ❤️s`}
```

Comment:
This element should live in `App.jsx`, at the highest level of the hierarchy of all the components. 

You can see an example of how this looks in App.jsx on line 95 [here](https://github.com/Ada-C22/flasky-front-end/commit/db8e1afd6d2e4cadf627841adcb1baec55544857#diff-d274a54187c91ba0f532df2a9e194e27ab50e988f5e4c33f5a7893918320c661R95) in flasky-front-end